dashboard/modules/actor/actor_head.py

```diff
@@ -230,6 +230,11 @@ async def kill_actor(self, req) -> aiohttp.web.Response:
 
         return rest_response(success=True, message=f"Killed actor with id {actor_id}")
 
+    @routes.get("/api/v0/actors")
+    async def get_actors(self, req) -> aiohttp.web.Response:
```

Comment:
How are we supporting list options like timeout and limit?

Reply:
I haven't added the code yet (sorry my bad). limit will be used for post-processing (after aggregation), and timeout will be used to decide the internal RPC timeout.